style.css

```diff
@@ -0,0 +1,559 @@
+.body {
+  max-width: 1920px;
+  min-width: 320px;
+  margin: 0 auto;
+  background: url('pic/iswanto-arif-448003-unsplash.png') no-repeat;
+  background-size: cover;
+  overflow: visible;
+  height: 100vh;
+}
+
+.header {
+  width: 100%;
+  padding: 87px 114px 0 199px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 0 0 130px 0;
+}
+
+.header::after {
+  content: "";
+  position: absolute;
+  height: 100vh;
+  width: 100vw;
+  background-color: #fff;
+  top: -100vh;
+  left: 0;
+  z-index: 2;
+  transition: .3s ease;
+}
+
+.container {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  margin-right: 4px;
+  letter-spacing: .25pt;
+}
+
+.nav__link {
+  color: #fff;
+  text-decoration: none;
+  font-size: 22px;
+  line-height: 26px;
+  font-family: 'Museo', sans-serif;
+  border-bottom: 1px solid transparent;
+  transition: .3s ease;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+.nav__link:not(:last-child) {
+  margin: 0 35px 0 0;
+}
+
+.nav__link:hover {
+  border-bottom: 1px solid #fff;
+  opacity: .7;
+}
+
+.nav__link:active {
+  opacity: 1;
+  transform: scale(.9);
+  transition: .1s ease;
+}
+
+.nav__link:focus {
+    outline: none;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)